front/app/components/SignUpIn/SignUp/index.tsx

```diff
@@ -55,6 +55,7 @@ const Container = styled.div`
   width: 100%;
   display: flex;
   flex-direction: column;
+  max-width: 580px;
```

Comment:
This doesn't seem needed, so can be removed to save the next person some time checking. :-)